jekyll/_cci2/android-machine-image.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: classic-docs
+title: "Using Android Images with the Machine Executor"
+short-title: "Android Image on the Machine Executor"
+description: "Using the Android Image on the Machine Executor"
+version:
+  - Cloud
+---
+
+## Overview
+
+The Android machine image is accessed through the [Linux `machine`
+executor]({{site.baseurl}}/2.0/configuration-reference/#machine-executor-linux),
+like other Linux machine images on CircleCI. The Android machine image supports
+nested virtualization and x86 Android emulators, so it can be used for Android
+UI testing. It also comes with the Android SDK pre-installed.
+
+## Using the Android machine image
+
+It is possible to configure the use of the Android image in your configuration
+with [orbs]({{site.baseurl}}/2.0/orb-intro) as well as manually. Using the
+Android orb will simplify your configuration while more complex and custom
+configurations may benefit from manually configuring your usage. This document
+will cover both use cases. Please view the [examples](#examples) section
+below for more details.
+
+## Pre-installed Software
+
+The Android machine image comes with the following pre-installed:
+
+### Android SDK
+- sdkmanager
+- Android platform 27, 28, 29, 30
```

Comment:
As we are doing an update to the image, can this be changed to `Android platform 23, 24, 25, 26, 27, 28, 29, 30, S` ?